pages/08.community-team/chapter.md

```diff
@@ -6,27 +6,34 @@ title: 'Community Team'
 
 ## Mission and expectations
 
-> We want to have more and more people who are contributing to Mautic.
+> We want more and more people to feel at home in the Mautic community. And we want to inspire & enable them to contribute to our cause.
 
 ### General Tasks
 
-* Grow the Community - global and local engagement
-* Increase active contribution
-* Facilitate communication and contribution
+* Support growth of the global as well as the local communities
+* Help the Mautic community be most ethical, accessible and inclusive
+* Provide really easy starting points for contribution
+* Organize and support Mautic events
+* Continue to improve community infrastructure
+* Connect with other Open Source communities
+* Work towards a Mautic NGO (e.g. „Mautic Association“)
```

Comment:
```suggestion
```

I am not sure this is on the cards for some time, we said we would review whether there was a need for such an entity annually:

> Therefore, we propose to revisit this discussion in twelve months (September 2020) in the context of the Community Council to determine whether the community is at a point where an NGO would be of benefit.  At this point, if it is determined by the Community Council that an NGO should be established, Acquia will fully support any processes necessary to enable its creation.
_Source: https://www.mautic.org/blog/press/response-to-community-consultation-on-the-governance-model-proposal_

So on that basis I would not feel comfortable with the wording of 'working towards a Mautic NGO' seeing as we have not even established a need for one in the Community Council yet.

Open to your thoughts on this topic.